alicloud/resource_alicloud_instance.go

```diff
@@ -121,6 +121,55 @@ func resourceAliyunInstance() *schema.Resource {
 				Default:      40,
 				ValidateFunc: validateIntegerInRange(40, 500),
 			},
+			"data_disks": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				MinItems: 1,
+				MaxItems: 15,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
```

Comment:
Please add validfunc for `name` and `description` based on the existing resource `alicloud_disk`